frontend/src/components/PlayRanking/PlayRanking.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import PlayRanking from '../PlayRanking/PlayRanking'
+
+export default {
+  title: 'PlayRanking',
+  component: PlayRanking,
+}
+
+export const playRanking = () => <PlayRanking play={play} />
```

Comment:
Ich weiß jetzt nicht genau, was 'play' eigentlich is... ich nehme mal an ein Objekt? Wenn ja, dann könntest du noch eine Story schreiben, die zeigt, wie das Ranking aussieht, wenn play ein leeres Objekt ist.

Reply:
Danke für den Hinweis. Ich muss mich noch genauer in Storybook einlesen ;) Play bezeichnet eine Spielepartie. Auf der BoardGameGeek-Seite verwenden sie auch diesen Begriff. Daher habe ich mich daran orientiert.